app/src/js/components/Requests/request.js

```diff
@@ -265,10 +327,21 @@ class RequestOverview extends React.Component {
     const { canEdit } = formPrivileges(this.props.privileges);
     const { roles } = this.props;
     const role = roles ? Object.keys(roles).map(role => roles[role].short_name) : [];
+    const { users } = this.props;
+    const filteredUsers = users.filter(user => user.user_roles.find(role => role.short_name == "uwg_member")
```

Comment:
For clarification here, you want the dropdown list to contain all UWG member regardless of DAAC association?

Reply:
All UWG members who have uwg_member role in edpub.